engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/runtime/RuntimeWithDisabledContextNode.java

```diff
@@ -13,14 +14,16 @@
     type = "Runtime",
     name = "with_disabled_context_builtin",
     description = "Disallows context in the specified scope.",
-    autoRegister = false)
+    autoRegister = false,
+    needsFrame = false)
```

Comment:
I still find this confusing. This node has `VirtualFrame` in its `execute` method, and at the same time it has `needsFrame = false` in its annotation. The behavior is not specified anywhere, looking at the generated code, it is clear that this node is simply inlineable. Can't we rename `needsFrame` to `forceInlining`? That is much clearer in my opinion.

Reply:
Right, `needsFrame` is confusing. However I am not sure whether `forceInlining` has clearer meaning? I could live with:
- `needsOwnFrame=false` - as that's what the option does - it decides whether the builtin gets _shared_ or _own_ frame
- `inlineable=true` - at the end this option decides whether we implement `InlineableNode.Root` or not

Reply:
I like `inlineable`. That's very clear. Probably more than both `needsFrame` and `forceInlining`.

Reply:
Be it `inlineable` then! c1c81a4